lighthouse-core/audits/third-party-summary.js

```diff
@@ -41,21 +37,31 @@ const PASS_THRESHOLD_IN_MS = 250;
 /** @typedef {import("third-party-web").IEntity} ThirdPartyEntity */
 
 /**
- * @typedef {{
- *   mainThreadTime: number,
- *   transferSize: number,
- *   blockingTime: number,
- * }} Summary
+ * @typedef Summary
+ * @property {number} mainThreadTime
+ * @property {number} transferSize
+ * @property {number} blockingTime
```

Comment:
There are a lot of style changes in this file with no accompanying functional changes anymore. Is it worth keeping the style changes anyway?

Reply:
these changes are good, lets keep